source/index.js

```diff
@@ -2,15 +2,22 @@ var Pixi = require("pixi.js")
 var Afloop = require("afloop")
 var Keyb = require("keyb")
 
-var WIDTH = 230, HEIGHT = 130
+import Player from "./player.js"
 
-var renderer = Pixi.autoDetectRenderer(WIDTH, HEIGHT)
+window.GAME_WIDTH = 230
+window.GAME_HEIGHT = 130
```

Comment:
+1 for `Reference.js`; much more descriptive name than just `Game.js`, which could be anything.